<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Welcome_screen</class>
 <widget class="QDialog" name="Welcome_screen">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome screen</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(11, 95, 157);</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>393</width>
     <height>106</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_welcome">
      <property name="styleSheet">
       <string notr="true">font: 36pt &quot;.AppleSystemUIFont&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Josh's Computer</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="Button_enter">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom-color: rgb(22, 151, 222);border-top-color: rgb(22, 151, 222);border-left-color: rgb(22, 151, 222);border-right-color: rgb(22, 151, 222);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
